server/src/service/matcha.ts

```diff
@@ -186,4 +186,16 @@ const sortMatchas = (matchas) => {
   })
 }
 
-export { getMatchaJobById, getMatchaJobs }
+const addMatchaDetailView = async (jobId: string) => {
+  await incrementOffre(jobId, {
+    stats_detail_view: 1,
+  })
+}
```

Comment:
Return type manquant + tsdoc:

```suggestion
const addMatchaDetailView = async (jobId: string) => incrementOffre(jobId, {
    stats_detail_view: 1,
  })
```

Reply:
c'est plutôt une bonne pratique de laisser le type de retour inféré par le compilateur. Ca permet d'avoir des types plus expressif. 
ok pour la doc